wear/src/main/java/com/android/gpstest/wear/MainActivity.kt

```diff
@@ -214,70 +219,48 @@ fun WearApp(signalInfoViewModel: SignalInfoViewModel) {
-                                satelliteMetadata.numSatsInView,
-                                satelliteMetadata.numSatsTotal
-                            )
+                        text = String.format("Long: ") + formatLatOrLon(
+                            Application.app,
+                            location.longitude,
+                            CoordinateType.LONGITUDE,
+                            prefs
                         )
                     )
                 }
                 item {
-                    if (location.provider.equals("invalid")) {
-                        Text(text = String.format("Bearing:"))
-                    } else {
-                        Text(text = String.format("Bearing: %.1f", location.bearing))
-                    }
+                    Text(
+                        text = "# Sats: " + formatSats(context, satelliteMetadata)
+                    )
                 }
                 item {
-                    if (dop.positionDop.isNaN()) {
-                        Text(text = "PDOP:")
-                    } else {
-                        Text(
-                            text = String.format(
-                                "PDOP: %s",
-                                stringResource(R.string.pdop_value, dop.positionDop)
-                            )
+                    Text(
+                        text = "Bearing: " + formatBearing(
+                            Application.app,
+                            location
                         )
-                    }
+                    )
                 }
                 item {
-                    if (dop.horizontalDop.isNaN() || dop.verticalDop.isNaN()) {
-                        Text(text = "H/V DOP:")
-                    } else {
-                        Text(
-                            text = String.format(
-                                "H/V DOP: %s", stringResource(
-                                    R.string.hvdop_value, dop.horizontalDop,
-                                    dop.verticalDop
-                                )
-                            )
-                        )
-                    }
+                    Text(text = "PDOP: " + formatDOP(context, dop))
                 }
                 item {
-                    if (location.provider.equals("invalid")) {
-                        Text(text = String.format("Speed:"))
-                    } else {
-                        Text(text = String.format("Speed: %.1f", location.speed))
-                    }
+                    Text(text = "H/V DOP: " + formatHVDOP(context, dop))
+                }
+                item {
+                    Text(
+                        text = "Speed: " + FormatUtils.formatSpeed(
+                            context, location, prefs
+                        )
+                    )
                 }
                 item {
                     StatusRowHeader(isGnss = true)
```

Comment:
I believe `StatusRowHeader` is duplicated across the app and wear and can be moved to library too? And I think the same for most (all?) of the other Status `@Composable` functions, like `StatusLabel` and `StatusValue`, `Cn0`, `Svid`, etc.

Reply:
Per offline discussion, the wear and app Composable functions for Text, etc. actually have different dependencies, so we're going to leave them for now.